packages/generator-emakinacee-react/generators/app/templates/shared/dot/.eslintrc

```diff
@@ -1,7 +1,10 @@
 {
     "extends": [
         "emakinacee-base",
-        "emakinacee-react"
+        "emakinacee-react",
+        // Must be last to override rules. https://github.com/prettier/eslint-config-prettier#installation
+        "prettier",
+        "prettier/react"
```

Comment:
These prettier configs, do they simply disable the rules mentioned at https://github.com/prettier/eslint-config-prettier#special-rules or will they overwrite anything else?

I was also considering if it would be nicer to align our emakinacee configs with prettier instead of using those prettier configs in addition.